pyproject.toml

```
@@ -3,7 +3,7 @@ line-length = 100
 
 [tool.poetry]
 name = "dbt-copilot-tools"
-version = "0.1.110"
+version = "0.1.112"
```

Why has this been bumped up two versions? 🤔 

```suggestion
version = "0.1.111"
```

---

Because I have another PR in front of this one...